arangod/Agency/MoveShard.cpp

```diff
@@ -674,11 +722,19 @@ JOB_STATUS MoveShard::pendingLeader() {
                          size_t found = 0;
                          for (size_t i = 1; i < plan.length() - 1; ++i) {
                            VPackSlice p = plan[i];
-                           for (auto const& c : VPackArrayIterator(current)) {
-                             if (arangodb::basics::VelocyPackHelper::equal(p, c, true)) {
-                               ++found;
-                               break;
+                           if (current.isArray()) { // done is not incremented, we'll remain pending
```

Comment:
the comment should read "found is not incremented".

Reply:
true

Reply:
true